test/integration/node-specific/custom_promise_library.test.js

```diff
@@ -9,8 +9,22 @@ class CustomPromise extends Promise {}
 CustomPromise.prototype.isCustomMongo = true;
 
 describe('Optional PromiseLibrary', function () {
+  beforeEach(() => {
+    PromiseProvider.set(null);
+  });
+
   afterEach(() => {
+    PromiseProvider.set(null);
+  });
+
+  it('should initially be set to null', () => {
```

Comment:
It's hard thing to test, upon loading the node process when our driver is first imported it should be set to null, but I'm not sure where I can put a test to make sure it's run "early", so it's intended to test the driver but it does always pass because of the beforeEach

Reply:
Just move it into a place where there isn't a before each, maybe?